manuscript/conclusion.md

```diff
@@ -81,11 +83,67 @@ As we saw in this book Open Source doesn't mean just release the code but let pe
+There are public projects like templates for WordPress, Guidelines (with web version on ReadTheDocs), mobile apps, SDK, design kits and others.  
+
+This was something that 5 years ago maybe for a Hard Sci-Fi author was impossible to think of. Like also the fresh news that [Inps, Inal and Istat are working together to create in 2023 a unique software house for their need](https://www.wired.it/article/inps-inail-e-istat-stanno-fondando-una-software-house/) (INPS is the national insitute for retirement, Inail the national insitute for work injuries and Istat the national insitute for statistics).
+
+Two things are very interesting for the purpose of this book about what they are doing.
+
+### PublicCode.yml standard
+
+`
+A metadata description standard for public software that is easy to use both for developers and people with less technical background in order to make the software developed by Public Administrations and Public Organizations easily discoverable. 
+`
+
+In few words, a file that you can put in your git repository that is like an identity card that allow to index it with references to the maintainer and dependencies.  
+The idea is to improve the discoverability and if that project is suitable for a public administration and to use the standard also in other countries.
+
+## Catalogo del software di Riuso
+
+That in English "Reuse Software catalog" that Italian public administration can use it. As every big organization departments don't talk each others, also cities or region, but they have often the same needs, so why develop multiple times the same software?  
+
+This is the idea at the end, a way for public administration where often managers have no so much knowledge on IT to find software already available and open source developed by themselves or by others like LibreOffice for example.  
+
+This project uses the standard we saw to fetch the software list, but the website is more friendly as includes more information, not just a description but people that you can reach to ask question. It is something that for a bureaucratic organization is important and that in OSS is often ignored.  
+After all, if you are maintaining a project you don't want that people reach you for questions about if the software can do this or that, or if you can implement something just for them. Especially if it is someone that maybe will pay you...
+
+This is an example about how improve the awareness to organization that is not like the usual software alternative list, because can give more information that are valuable to the people in charge of decisions. 
```

Comment:
```suggestion
This is an example about how to improve the awareness for organization that is not like the usual software alternative list, because it can give more information that are valuable to the people in charge of decisions. 
```